.mailmap

```diff
@@ -2,3 +2,5 @@ Bane Sullivan <banesullivan@gmail.com> banesullivan <banesulli@gmail.com>
 
 Alex Kaszynski <akascap@gmail.com> Alex Kaszynski <akaszynski@users.noreply.github.com>
 Alex Kaszynski <akascap@gmail.com> akaszynski <akascap@gmail.com>
+
+Andras Deak <deak.andris@gmail.com> Andras Deak <adeak@users.noreply.github.com>
```

Comment:
Does this fix the history locally or do you also need:
```
Andras Deak <deak.andris@gmail.com> Andras Deak <noreply@github.com>
```

Reply:
All my existing commits up to master (which is not a lot) have the noreply address. I've searched my name in the log of master and each seems to have been fixed by this.

Reply:
From a quick search nobody has `noreply@github.com` in the log.

Reply:
Okay then!